src/test/compile-fail/pat-range-bad-dots.rs

```diff
@@ -8,11 +8,9 @@
 // option. This file may not be copied, modified, or distributed
 // except according to those terms.
```

Comment:
Note: with #38914 merged in the meantime, you'll either have to rename this test to "feature-gate-exclusive-range-pattern.rs" or add a comment `// gate-test exclusive_range_pattern`. See also [COMPILER_TESTS.md](https://github.com/rust-lang/rust/blob/master/COMPILER_TESTS.md#summary-of-header-commands). Just saying as changing beforehad will prevent a failed bors run and maybe contribute to getting this merged faster.